components/core/cmake/Modules/FindMariaDBClient.cmake

```
@@ -20,6 +20,25 @@ include(cmake/Modules/FindLibraryDependencies.cmake)
 find_package(PkgConfig)
 pkg_check_modules(mariadbclient_PKGCONF QUIET "lib${mariadbclient_LIBNAME}")
 
+if(NOT mariadbclient_PKGCONF_FOUND AND APPLE)
+    execute_process(
+        COMMAND brew --prefix mariadb-connector-c
+        RESULT_VARIABLE brew_result
+        OUTPUT_VARIABLE mariadbclient_MACOS_PREFIX
+        ERROR_QUIET
```

Might be better if we don't hide any error messages.